content/posts/detect-sensitive-data-with-the-nightfall-filter-plugin.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Detect Sensitive Data With the Nightfall Filter Plugin'
+date: "2022-04-04"
+description: "Learn how to set up the Nightfall filter plugin to start detecting sensitive data in your logging pipeline"
+image: "/images/Detect Sensitive Data With the Nightfall Filter Plugin.png"
+author: "Victor Chen"
+
+---
+
+##### Introduction
+
+Sensitive data often makes its way into [logs](https://nightfall.ai/how-does-sensitive-information-end-up-in-observability-platforms). Though most of the time unintentional, these incidents have the potential to do a lot of damage, as they usually involve exposure of API keys, passwords, and customer data that can give attackers access to critical business data or put companies at compliance risk. This is why it’s important for organizations to be proactive about securing their logs. Because [identifying sensitive data](https://nightfall.ai/why-detecting-sensitive-data-is-hard) is not an easy task, Nightfall built a plugin for Fluent Bit. In this tutorial, we will go over how to set up Fluent Bit and the Nightfall plugin to process, forward, and secure your logs.
+
+##### Setting Up Your Nightfall Developer Platform Policy
+
+If you do not have a Nightfall account, you can sign up for free [here](https://app.nightfall.ai/sign-up). Once you have an account, navigate to Detection Engine > Detection Rules where you can create a [detection rule](https://help.nightfall.ai/detection/getting-started-with-the-detection-engine) with which you can secure your logs. Nightfall defines a detection rule as an aggregation of detectors: for example, ```{“credit card number” or “social security number”}```.
+
+![detection-rule-page](/images/blog/blog-nightfall-detection-rule-page.png)
+
+A good starting detection rule for scanning your logs could include the API key detector, database connection string detector, and cryptographic key detector, like so:
+
+![detection-rule-creation](/images/blog/blog-nightfall-detection-rule-creation.png)
+
+After you have created your detection rule(s), navigate to Developer Platform > Overview where you can create an API key and a Developer Platform Policy that will power the Nightfall plugin. Nightfall uses policies to describe how our platform should respond when a detection rule violation is triggered.
+
+![dev-platform-page](/images/blog/blog-nightfall-dev-platform-page.png)
+
+Under the API Keys tab, create an API key, which you will need to run the Nightfall plugin. Under the Policies tab, create a policy with the detection rule(s) you created above. You can also configure where alerts from Nightfall about findings in your logs should be sent, featuring integrations like email, Slack, and HTTP Webhook events.
+
+##### Running Fluent Bit With the Nightfall Plugin
+
+###### Docker Image
+
+The Nightfall plugin is available in Fluent Bit versions after 1.9.1, so you can pull directly from the fluent bit repository:
```

Comment:
Overall this looks awesome, would you be able to add this to the top saying that the nightfall team contributed the plugin and it is now available in Fluent Bit 1.9.1?

Reply:
done!